docker-for-aws/index.md

```diff
@@ -133,7 +153,7 @@ Elastic Load Balancers (ELBs) are set up to help with routing traffic to your sw
 
 Docker for AWS automatically configures logging to Cloudwatch for containers you run on Docker for AWS. A Log Group is created for each Docker for AWS install, and a log stream for each container.
 
-`docker logs` and `docker service logs` are not supported on Docker for AWS. Instead, you should check container in CloudWatch.
+`docker logs` and `docker service logs` are not supported on Docker for AWS when using Cloudwatch for logs. Instead, you should check container logs in CloudWatch.
```

Comment:
The `docker logs` and `docker service logs` commands are not supported on... Instead, check container logs...